src/applications/mhv-medical-records/sass/print-download.scss

```diff
@@ -15,6 +15,10 @@
   button:focus {
     background-color: var(--vads-color-gray-cool-light);
     color: var(--vads-color-primary-dark);
+    /* 3. Ensure the element is positioned and has a high z-index */
```

Comment:
I think this solution is good, but I feel like the number "3." in the comment got left in here by accident. :)